src/input/input_json.rs

```diff
@@ -115,23 +115,29 @@ impl<'a> Input<'a> for JsonInput {
         }
     }
 
-    fn strict_float(&self) -> ValResult<f64> {
+    fn strict_float(&self, _only_finite: bool) -> ValResult<f64> {
```

Comment:
doesn't make sense for strict mode of JSON since a string is required to represent not finite values